### PR TITLE
Feature/visibility

### PR DESCRIPTION
**Pull request by DzmitryI**

### CODE REVIEW

File: src/components/membersGrid/MembersGrid.js [hunk 1]
```diff
@@ -61,41 +63,62 @@ class MembersGrid extends Component {
...
+    const { directions, fetchMembersSuccess } = this.props;
+    const dragRow = members[dragIndex];
+    members = update(members, {
+      $splice: [
+        [dragIndex, 1],
+        [hoverIndex, 0, dragRow],
+      ],
+    });
+    fetchMembersSuccess(members, directions);
+  };
```
Comment (maximsan) [inline src/components/membersGrid/MembersGrid.js:75]: I believe you can make common method `moveRow` and use it everywhere

File: src/components/taskTracksGrid/TaskTracksGrid.js [hunk 2]
```diff
@@ -97,31 +113,39 @@ class TaskTracksGrid extends Component {
...
+                              <Button
+                                className='btn-delete'
+                                onClick={this.onDeleteClick}
+                                name='Delete'
+                                disabled={adminMentor}
+                              />
+                            </>
+                          }
+                        />
+                      </>
```
Comment (maximsan) [inline src/components/taskTracksGrid/TaskTracksGrid.js:146]: move to separate components that you use everywhere as `value`

File: src/index.js [hunk 3]
```diff
@@ -15,10 +17,12 @@ const composeEnhancers =
...
-      <App />
-    </BrowserRouter>
-  </Provider>
+  <DndProvider backend={HTML5Backend}>
+    <Provider store={store}>
+      <BrowserRouter>
+        <App />
+      </BrowserRouter>
+    </Provider>
+  </DndProvider>
```
Comment (maximsan) [inline src/index.js:26]: I will propose you to move dndprovider inside redux provider

Comment (DzmitryI) [reply src/index.js:26]: fixed

File: src/components/app/App.js [hunk 4]
```diff
@@ -49,17 +48,12 @@ class App extends Component {
...
 
   async componentDidUpdate(prevProps) {
-    const { email, database } = this.props;
-    let fetchService = {};
+    const { email } = this.props;
+    let fetchService = new FetchFirebase();
     if (email && email !== prevProps.email) {
       if (!email) {
         this.setState({ theme: 'light' });
       }
```
[the remark this answers is not in the document]
Comment (DzmitryI) [reply src/components/app/App.js:56]: when we'll be logout, it'll be true

**Pull request merged**